### PR TITLE
Adjust mail entries

### PR DESCRIPTION
**Pull request by LukasReschke**
This adjusts the mail entries to the private addresses of Arthur, Björn, Frank and Lukas.

Since I already was updating my address I asked them whether I should change theirs as well and they agreed.

I also fixed some other duplicates that were new.

cc @karlitschek @blizzz @schiesbn

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -16,8 +16,9 @@ Andreas Schönebeck <aschoenebeck@web.de>
 Andreas Ergenzinger <andreas.ergenzinger@gmx.de> AndreasErgenzinger <andreas.ergenzinger@gmx.de>
 Andrew Brown <andrew@casabrown.com>
 André Gaul <gaul@web-yard.de>
-Arthur Schiwon <blizzz@owncloud.com> Arthur Schiwon <blizzz@arthur-schiwon.de>
-Arthur Schiwon <blizzz@owncloud.com> blizzz <blizzz@owncloud.com>
+Arthur Schiwon <blizzz@arthur-schiwon.de> Arthur Schiwon <blizzz@owncloud.com>
+Arthur Schiwon <blizzz@arthur-schiwon.de> blizzz <blizzz@owncloud.com>
+Arthur Schiwon <blizzz@arthur-schiwon.de> Arthur Schiwon <blizzz@owncloud.com>
```
Comment (nickvergessen) [inline .mailmap]: duplicate?

**Comment by nickvergessen** [comment]
Otherwise :+1:

**Comment by grdfsfgdggsf** [comment]
thanks 👍

**Comment by georgehrke** [comment]
👍

**Comment by georgehrke** [comment]
@LukasReschke please rebase

File: .mailmap (hunk 1 above)
Comment (blizzz) [inline .mailmap]: Seems so, works otherwise.

**Comment by LukasReschke** [comment]
Rebased

**Pull request merged**